src/ORM/FieldType/DBBoolean.php

```diff
@@ -2,22 +2,26 @@
 
 namespace SilverStripe\ORM\FieldType;
 
+use SilverStripe\Core\Validation\FieldValidation\BooleanFieldValidator;
 use SilverStripe\Forms\CheckboxField;
 use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\FormField;
 use SilverStripe\ORM\DB;
 use SilverStripe\Model\ModelData;
 
 /**
- * Represents a boolean field.
+ * Represents a boolean field
+ * Values are stored in the database as tinyint i.e. 1 or 0
```

Comment:
I don't think that's a detail that's really needed here, but I won't block merging if you leave it in.
```suggestion
 * Represents a boolean field
```

Reply:
Leave as is, it's a pretty important detail because the conversion to bool to int is important to be aware of

Reply:
I don't think it is an important detail because that's something that happens just before it gets sent to the DB which is abstracted away and if they _do_ need to go that deep in, they can see it in the implementation. But I'm not gonna bother arguing about this, so leave it.